src/net/peer/connect/bootstrap/cache.rs

```diff
@@ -43,6 +42,6 @@ impl Cache {
         self.file_handler.read_file().ok().unwrap_or_else(|| vec![])
     }
 
-    pub fn remove_peer_acceptor(&mut self, _peer: SocketAddr) {}
+    //pub fn remove_peer_acceptor(&mut self, _peer: SocketAddr) {}
```

Comment:
Why not just remove instead of commeting?

Reply:
I've removed it now.